R/auto.R

```diff
@@ -140,22 +200,59 @@ get_remote_name <- function(branch = get_main_branch()) {
   remote
 }
 
-create_release_branch <- function(force) {
-  branch_name <- paste0("cran-", desc::desc_get_version())
+merge_dev_news <- function(fledgeling, new_version) {
+  dev_idx <- grepl("^[0-9]+[.][0-9]+[.][0-9]+[.][0-9]+$", fledgeling$news$version)
+  stopifnot(dev_idx[[1]])
```

Comment:
why? add comment?

Reply:
Otherwise, we would be counting the wrong thing in the `rle()` call below. A comment would be great indeed, or perhaps extract a function?